README.md

```diff
@@ -0,0 +1,32 @@
+# termcord
+
+[![Tests](https://github.com/haguro/termcord/actions/workflows/tests.yml/badge.svg)](https://github.com/haguro/termcord/actions/workflows/tests.yml) [![Golint](https://github.com/haguro/termcord/actions/workflows/golint.yml/badge.svg)](https://github.com/haguro/termcord/actions/workflows/golint.yml) [![Go Report Card](https://goreportcard.com/badge/github.com/haguro/termcord)](https://goreportcard.com/report/github.com/haguro/termcord)
+
+`termcord` is a terminal session recorder written in Go.
+
+## Installation
+1. Install Go.
```

Comment:
It might be fun to think about ways to make the binary available without people having to compile it themselves. Of course, all sensible people already have Go installed, and so will people who want to use your _library_, but people interested in the _tool_ might just want the tool, mightn't they? Some possible options for this include GitHub releases, for example.